### ISSUE TITLE
TreeChain draw fails: 'Graph' object has no attribute 'SetDirectory' 

### ISSUE THREAD

**Issue by Munken** [rootpy/rootpy#741]
When executing a 2D draw command (`chain.Draw("a:b")`) with TreeChain it fails with the following message:

```
Traceback (most recent call last):
  File "angular.py", line 8, in <module>
    tData.Draw("uE:dE", "")
  File "/home/munk/.local/lib/python2.7/site-packages/rootpy-0.8.0.dev0-py2.7.egg/rootpy/tree/chain.py", line 119, in Draw
    output.SetDirectory(0)
AttributeError: 'Graph' object has no attribute 'SetDirectory'
```

The caveat is, that unlike histograms, graphs is not associated with the current directory.

**Comment by ndawe** [comment]
For the moment I'd suggest sticking with ROOT.TChain for Draw()-ing. rootpy's TreeChain is mainly beneficial when looping entry-by-entry. TreeChain's Draw can only handle histograms currently.

**Comment by ndawe** [comment]
Fixed in #750

**Comment by ndawe** [comment]
TreeChain now handles Graphs too.
